src/index.js

```diff
@@ -25,17 +25,22 @@ class RetryChunkLoadPlugin {
           }
         `;
 
-          const cacheBust = this.options.cacheBust
-            ? `
-          (${this.options.cacheBust})();
-        `
-            : '"cache-bust=true"';
+          const getCacheBustString = () =>
+            this.options.cacheBust
+              ? `
+            (${this.options.cacheBust})();
+          `
+              : '"cache-bust=true"';
+
+          const maxRetries = Number.isInteger(Number(this.options.maxRetries))
```

Comment:
This checks if the value from this.options.maxRetries is an integer.
If it is a floating number or undefined or null then it uses the default value, i.e. 1.

Reply:
The number should be greater than 0 or retries would become infinity

Reply:
True.
Have updated the code to have a check for it to be greater than 0